models/transfers/bnb/bep20/transfers_bnb_bep20.sql

```diff
@@ -0,0 +1,99 @@
+{{ config(
+    materialized='view',
+    alias='bep20',
+    post_hook='{{ expose_spells(\'["bnb"]\',
+                                    "sector",
+                                    "transfers",
+                                    \'["hosuke"]\') }}')
+}}
+
+
+with
+    sent_transfers as (
+        select
+            'send' || '-' || evt_tx_hash || '-' || evt_index || '-' || `to` as unique_transfer_id,
```

Comment:
We are not using this column downstream if I checked correctly? In that case, it can be removed here and also in the CTEs below.

Reply:
This CTE is used in line 66, the unused `sent_transactions` and `received_transactions` have been removed.

Reply:
Sorry if I was not clear. I meant removing the `unique_transfer_id` column (not the CTE itself) in this line and all other occurrences in this script.

Reply:
Sorry I just got the point that `unique_transfer_id` is unused at the moment. I have removed this column in the latest commit. Thank you for your patience in explaining